website/source/guides/external/lxc.html.md

```diff
@@ -136,37 +96,9 @@ Driver Status = docker,exec,java,lxc,mock_driver,raw_exec,rkt
 ...
 ```
 
-### Step 5: Generate a Job File
-
-Create a file named `lxc.nomad` and place the following contents in it:
-
-```hcl
-job "example-lxc" {
-  datacenters = ["dc1"]
-  type        = "service"
-
-  group "example" {
-    task "example" {
-      driver = "lxc"
-
-      config {
-        log_level = "trace"
-        verbosity = "verbose"
-        template  = "/usr/share/lxc/templates/lxc-busybox"
-      }
-
-      resources {
-        cpu    = 500
-        memory = 256
-      }
-    }
-  }
-}
-```
-
-### Step 6: Register the Nomad Job
+### Step 5: Register the Nomad Job
 
-Run the following command to register your Nomad job:
+Register a Nomad job that deploys an LXC workload and confirm the evaluation (you can find and example job [here][lxc-job]):
```

Comment:
Replace with "You can run this [LXC example job] to register a Nomad job that deploys an LXC workload" for better linkification.